master/internal/kubernetes/request_queue.go

```diff
@@ -47,7 +47,9 @@ type (
 
 // message types sent from requestProcessingWorkers to requestQueue.
 type (
-	workerAvailable struct{}
+	workerAvailable struct {
+		resourceHandler *actor.Ref
```

Comment:
blocking: it seems to be accurate to let this name be `requestFinished` and put `queuedResourceRequest` in it if it's necessary.